src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -28,12 +28,7 @@
 import java.security.InvalidKeyException;
 import java.security.NoSuchAlgorithmException;
 import java.text.DateFormat;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;

```

Comment:
This is probably IDE config making star import when you're using more than 5 imports.